### PR TITLE
I67 vignette cassandra

### PR DESCRIPTION
**Pull request by Cassandra291201**
Made changes to:
1. "Creating the ..." section
2. "Estimating Marginal ..." section

### CODE REVIEW

File: vignettes/mi_vignette/index.Rmd/.RData
Comment (josherrickson) [inline vignettes/mi_vignette/index.Rmd/.RData:1]: Thanks for taking a look @Cassandra291201 !

These files shouldn't be added. we should have added them to .gitignore much earlier - sorry about that! Please add .RData, .Rapp.history and .Rhistory to the .gitignore file in the main folder, and then remove these three files - they should no longer be automatically added.

Comment (Cassandra291201) [reply vignettes/mi_vignette/index.Rmd/.RData:1]: Sounds good, made a PR on the changes to .gitignore. And I will remove these three files. Thank you!

File: vignettes/mi_vignette/index.Rmd/index.Rmd
Comment (josherrickson) [inline vignettes/mi_vignette/index.Rmd/index.Rmd:1]: I think this file got moved (path says `vignettes/mi_vignette/index.Rmd/index.Rmd` with the double `index.Rmd`), which is likely causing the failed pkgdown build.

Comment (Cassandra291201) [reply vignettes/mi_vignette/index.Rmd/index.Rmd:1]: Hi Josh, thank you for reviewing my PR. I checked the file path but couldn't figure out what was wrong here. I submitted another pull request and it seems to work fine. I'll close this PR, and please refer to this new PR [here](https://github.com/benbhansen-stats/propertee/pull/201).

**Pull request closed without merging**